src/utils/get-set.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * @name Utils.defineGetterSetter
```

Comment:
I understand paradigms like the file `src/utils/get-ratio.js` which is accessible at `Utils.getRatio`, but this doesn't share that paradigm and the accessor is different from the previous version of Two.js. What's the thinking here?

Reply:
original location was `Two.Utils.defineProperty`

Reply:
I thought `defineGetterSetter` was a clearer name, since it has different behavior to `Object.defineProperty`.